src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -334,6 +334,7 @@ export const Details: React.FC<DetailsType> = ({
                     }
                 },
             )
+            .catch(noop)
```

Comment:
should we show a error here?

Reply:
Yes, the ui placeholders would show loading but the user wouldn't be notified of the failure.